config/default.json

```diff
@@ -73,6 +73,11 @@
             "location": "autoAcceptPendingRelationships/AutoAcceptPendingRelationshipsModule",
             "enabled": false
         },
+        "autoDecomposeDeletionProposedRelationships": {
+            "displayName": "Auto Decompose DeletionProposed Pending Relationships",
```

Comment:
Omit the "Pending" in the `"displayName"`: "Auto Decompose DeletionProposed ~~Pending~~ Relationships"

Reply:
check!